src/Parser/MarksmanshipHunter/Modules/Features/TimeFocusCapped.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Icon from 'common/Icon';
+
+import FocusTracker from '../FocusChart/FocusTracker';
+
+class TimeFocusCapped extends Module {
+  static dependencies = {
+    focusTracker: FocusTracker,
+  };
+
+  getTotalWaste(){
+    let waste = this.focusTracker.secondsCapped * this.focusTracker.focusGen;
+    if (this.focusTracker.generatorCasts) {
+      Object.keys(this.focusTracker.activeFocusWasted).forEach((generator) => {
+        waste += this.focusTracker.activeFocusWasted[generator];
+
+      });
+    }
+    return Math.round(waste);
+  }
+
+  statistic() {
+    const totalFocusWaste = this.getTotalWaste();
+    const tooltipData = "You wasted <b>" + totalFocusWaste + " </b> focus. <br /> That's <b>" + Math.round(totalFocusWaste/(this.owner.fightDuration/1000 * this.focusTracker.focusGen) * 10000)/100 + "% </b> of your total focus generated. <br /> For more details, see the Focus Chart tab.";
+    const percentCapped = Math.round(this.focusTracker.secondsCapped/(this.owner.fightDuration/1000) * 10000)/100;
+    return (
+      <StatisticBox
+        icon = {<Icon icon='ability_hunter_focusfire' alt = 'Focus Wasted' />}
+        label = {'Time Focus Capped'}
+        tooltip= {tooltipData}
+        value =  {percentCapped + " %"}
+      //Time not Focus-Capped: {Math.round((this.owner.fightDuration / 1000 - this.focusTracker.secondsCapped) * 100) / 100}s / {Math.floor(this.owner.fightDuration / 1000)}
+      footer={(
+        <div className="statistic-bar">
+        <div
+          className="stat-health-bg"
+          style={{ width: `${(100-percentCapped)}%` }}
+          data-tip={`You spent <b>${100-percentCapped}%</b> of your time, or <b>${Math.round(Math.floor(this.owner.fightDuration/1000) - this.focusTracker.secondsCapped)}s</b> under the focucs cap.`}
+        >
+          <img src="/img/sword.png" alt="Time not focus capped" />
```

Comment:
I think this icon should be removed or replace by something more fitting. It does not appear relevant to the shown information.

Reply:
I think it's relevance is a lot more clear to MMS hunters than it would be to other classes. While the icons might seem somewhat irrelevant with a lot of classes, the way in which we use focus makes them relevant when looking at the fight from the viewpoint of spenders = damage. Even our most common focus spenders generally do 5-10x more damage per cast when compared to our generators, and generators quite often make up less than 5% of our total damage in a fight. From this viewpoint, there are only 2 things happening when you're focus capped. Either you're actually "afk", or you're effectively "afk", ie only casting generators, which are incredibly weak in terms of damage and pretty much useless when you're already focus capped. Because this is the mindset that MMS hunters view their abilities, I think it's surprising appropriate to use a sword (symbolizing attacking) and an afk (symbolizing not attacking) icon, since our class finds very little benefit from non-focus spending abilities.

Reply:
The sword/afk icons are meant for the downtime statistic, and using them in two places to mean different things would be confusing, even if they might be appropriate enough.